internal/retry/retryable_errors.go

```diff
@@ -18,38 +17,49 @@ import (
 var (
 	_ basetypes.ObjectTypable  = RetryType{}
 	_ basetypes.ObjectValuable = RetryValue{}
+
+	defaultRetryableStatusCodes                = []int{429}
```

Comment:
I think 429 retry is already covered by the azure SDK?

Reply:
This acted as a safety net, but I agree probably not necessary given the new provider level configuration.